labshare-client/src/App.vue

```diff
@@ -51,10 +51,11 @@
             </template>
 
             <b-nav-item-dropdown v-if="$authenticated" right :text="userName">
-              <b-dropdown-item href="#/profile">{{ $t("profile") }}</b-dropdown-item>
+              <b-dropdown-item href="#/profile" v-if="$user.role && $user.role.indexOf('dmin') == -1">{{ $t("profile") }}</b-dropdown-item>
```

Comment:
RE 1: actually no, because there exists "admin" and "superAdmin" :D But I should probably convert it to lower case first...

RE 2: Jep definitely, was too lazy for this...

Reply:
In that case, it is worth at least pulling this out into the script and looking up both `admin` and `superAdmin` explicitly by equality for now. The way this works currently is unreliable, as any status that has "dmin" would work. (In practice this is unlikely, but not impossible, and it's not something we want to ever happen)